RedHat-UX/red-hat-design-system
# docs: design/code status page updates

## What I did

1. Updated [Design/code status](https://ux.redhat.com/design-code-status/) page to reflect the progress in building the RHDS repo and to add/fix documentation links.

2. Also fixed this error in our component status tables:
![Screen Shot 2022-10-04 at 4 57 40 PM](https://user-images.githubusercontent.com/95588923/193926980-b48b8581-7c17-46fa-aaed-d2c94ab4913e.png)


## Testing Instructions

1. Visit https://deploy-preview-567--red-hat-design-system.netlify.app/design-code-status/ to view Design/code status page updates.
2. Visit any component page (like [Alert](https://deploy-preview-567--red-hat-design-system.netlify.app/components/alert/)) to check component status table.

## Notes to Reviewers
